src/sql/workbench/contrib/scripting/browser/scripting.contribution.ts

```diff
@@ -70,7 +70,7 @@ MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
 		id: commands.SCRIPT_AS_ALTER_COMMAND_ID,
 		title: localize('scriptAsAlter', "Script as Alter")
 	},
-	when: MssqlNodeContext.CanScriptAsAlter
+	when: ContextKeyExpr.or(MssqlNodeContext.CanScriptAsAlter, TreeNodeContextKey.NodeType.isEqualTo(NodeType.Function))
```

Comment:
it looks like NodeType.Function is already added to `canAlter`.  Any idea why it also needs to be special-cased here?